website/docs/docs/dbt-cloud/dbt-cloud-api/service-tokens.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Service Account and Metadata API tokens"
```

Comment:
I think the title here is just "Service Account API Tokens", right? Metadata-only API tokens are a type of Service Account tokens